src/datepicker/datepicker-input.spec.ts

```diff
@@ -202,6 +212,49 @@ describe('NgbInputDatepicker', () => {
        }));
   });
 
+  describe('without form control', () => {
+    it('should propagate disabled state', fakeAsync(() => {
```

Comment:
this test should not be asynchronous, you're not using `ngModel` anymore and no need in the additional `describe` I guess